wrapper.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+node usr/azure-integration/bridge.js
+printf 'INTEGRATION PROCESS FATAL ERROR: %s\n' "$?"
+
+exit 0
```

Comment:
Return the exit code here. I now see what this wrapper is for. It should only print the fatal error when the exit code != 0.

On a graceful shutdown (SIGTERM), the process should shutdown with exit 0 and this is not a fatal error. When updating or scaling down, an exit 0 is fine and should not initiate a crash report.